### ISSUE TITLE
[QUESTION] customized chemical mechanism

### ISSUE THREAD

**Issue by jinlx** [geoschem/geos-chem#1306]
Hi there, I am looking for docs to customize mechanism in GEOS-Chem but it seems that the previous doc has been deleted. Everything related to KPP is merged to https://kpp.readthedocs.io/en/latest/index.html but I am not able to find the previous doc on how to customize mechanism. I was wondering if it's easy to put previous manual in the new web or another place so that people can still easily follow the manual to change chemistry in GEOS-Chem .

Cheers,
Lixu

**Comment by yantosca** [comment]
Oops the link should be: https://geos-chem.readthedocs.io/en/latest/geos-chem-shared-docs/supplemental-guides/using-kpp-with-gc.html.

**Comment by jinlx** [comment]
Thanks for the quick reply Bob! I will close the issue for now!

**Comment by yantosca** [comment]
Thanks @jinlx.  Let me know if anything in the doc needs updating.  We are trying to update all the docs (KPP, GEOS-Chem, HEMCO, etc.) before 14.0.0 gets released.

Also note: 14.0.0 will have code generated by KPP 2.5.0, so you will need to update your KPP version if you plan on modifying mechanisms in 14.0.0 or higher versions.  We are currently in the benchmarking process for 14.0.0, so maybe a mid-Aug release.

**Comment by jinlx** [comment]
HI Bob! This is a follow-up question for adding new compound in the GEOS-Chem. I recently checked customized GEOS-Chem simulation with newly added compounds. The model runs smoothly and I can get those compounds information in GEOSChem.SpeciesConc.yyyymmdd_0000z.nc4. However, species concentration are all 0 for newly added comounds. Also those newly added compounds are not saved in HEMCO_diagnostics.yyyymmddhhmm.nc even though I have set up in the HEMCO_Diagn.rc. 

In order to get those new speices saved, I was wondering if we need to make any edition in input.geos (i.e., ADVECTED SPECIES MENU) and species_database.yml. 

Thanks,
Lixu

**Comment by jinlx** [comment]
HI @yantosca ! This is a follow-up question for adding new compound in the GEOS-Chem. I recently checked customized GEOS-Chem simulation with newly added compounds. The model runs smoothly and I can get those compounds information in GEOSChem.SpeciesConc.yyyymmdd_0000z.nc4. However, species concentration are all 0 for newly added comounds (emission information added). Also those newly added compounds are not saved in HEMCO_diagnostics.yyyymmddhhmm.nc even though I have set up in the HEMCO_Diagn.rc.

In order to get those new speices saved, I was wondering if we need to make any edition in input.geos (i.e., ADVECTED SPECIES MENU) and species_database.yml.

Thanks,
Lixu
